src/hooks/usePrevious/__stories__/usePrevious.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import usePrevious from "../index";
+import UsePrevious from "./hooksDummyComponents/UsePrevious";
+import Button from "components/Button/Button";
+import { Counter } from "components";
+import Flex from "components/Flex/Flex";
+import "./usePrevious.stories.scss";
+
+<Meta title="Hooks/usePrevious" component={UsePrevious} />
+
+# usePrevious
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+Hook for keeping previous state value.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const [count, setCount] = useState(1);
+      const prevCount = usePrevious(count);
+      const incrementButtonOnClick = () => {
+        setCount(prevValue => prevValue + 1);
+      };
+      return (
+        <div>
+          <Flex className="use-previous-counter-container" direction={Flex.directions.COLUMN}>
```

Comment:
let's use css modules for every new class because we planing to migrate everything to css modules is this q or the next one.

Reply:
also let's talk about how to name classes when using css modules :)